Adding Open IE and fixing dependency parsing

gabrielStanovsky opened the pull request:
- Add Open IE to the models page.
- Fix #74

DeNeutoy commented on models.html:
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
+                </div>
+                <div data-tab="training" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The Open Information extractor was evaluated on the <a href="https://github.com/gabrielStanovsky/oie-benchmark" target="_blank">OIE2016 corpus</a>.</p>
```
The data is freely available right? You should upload it to the allennlp s3 bucket and add a config https://github.com/allenai/allennlp/tree/master/training_config here which replicates the training proceedure for this model, if possible.

DeNeutoy commented on models.html:
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
+        <div class="constrained constrained--med">
+          <div class="padded-med">
+            <h2 id="open-information-extraction">Open Information Extraction</h2>
+            <p class="t-sm">
```
Can you add some performance statistics here?

gabrielStanovsky replied:
Done.

DeNeutoy commented on models.html:
```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
               </div>
             </div>
 
+      <!-- Dependency Parsing-->
```
Ah nice one, thanks.

gabrielStanovsky commented:
Thanks @DeNeutoy, I addressed your comments.
Regarding data availability - unfortunately it's not publicly available and requires LDC license (PTB texts).
I added a more explicit note on that.
